cylc/flow/network/ssh_client.py

```diff
@@ -45,11 +47,43 @@ def __init__(
             timeout: Union[float, str] = None
     ):
         self.workflow = workflow
-
+        self.SLEEP_INTERVAL = 0.1
         if not host:
             self.host, _, _ = get_location(workflow)
 
-    def send_request(self, command, args=None, timeout=None):
+    async def async_request(self, command, args=None, timeout=None):
+        """Send asynchronous request via SSH.
+        """
+        self.timeout = timeout if timeout else 300  # 5 min default timeout
```

Comment:
I think it would make more sense to move this to `self.__init__()` and instead do the same thing as the ZMQ client's `async_request()`
https://github.com/cylc/cylc-flow/blob/7377b3cffe57ac01102cbc77e0ae669266bd95cf/cylc/flow/network/client.py#L168

Reply:
(note `async_timeout` and the `--comms-timeout` interfaces take time in seconds not miliseconds)